tests/lmeval/test_lmeval.py

```diff
@@ -160,7 +163,11 @@ def _run_lm_eval(self):
             logger.info(
                 f"Comparing {metric}: Expected {expected_val}, Got {actual_val}"
             )
-            assert numpy.isclose(expected_val, actual_val, rtol=0.05)
+            if self.lmeval.use_stderr_atol:
+                std_err = metrics.get(metric.replace(",none", "_stderr,none"))
```

Comment:
nit: This is a little bit hard to understand

Reply:
I refactored the config and this logic to just use user-provided stderr vals, take a look and lmk what you think